.gitignore

```diff
@@ -27,3 +27,10 @@ newrelic_agent.log
 
 # approval tests
 *.received.*
+
+# cypress
```

Comment:
Maybe move this to a `.gitignore` inside the `cypress` directory to keep this specific stuff in one place (and make it easier if it moves again)